File: 7.multiplication.c
#include<stdio.h>  
int main()  
{
  int number;  
  printf("Enter Multiplication : "); 
  scanf("%d",&number);  
  int sum;
  if(number>12){
      number=12;
  }
  for(int i = 1; i <= number; i++){ 
    printf("*****************************\n");
    for(int num = 1; num <= 10; num++){ 
        printf("%d x %d = %d\n", i, num, i*num);
    } 
    printf("*****************************\n");
  } 
  printf("Result: %d", sum);  
}

File: 7.multiplication.cs
using System;

class Program{
    static void Main(){
      Console.WriteLine("Enter number: ");
      int number = Convert.ToInt32(Console.ReadLine());
      int sum = 0;
      if(number>12){
        number=12;
      }
      for(int i = 1; i <= number; i++){ 
        Console.WriteLine("*****************************");
        for(int num = 1; num <= 10; num++){ 
            Console.WriteLine($"{i} x {num} = {i*num}");
        } 
        Console.WriteLine("*****************************");
      } 
      Console.WriteLine($"Result: {sum}"); 
    }
}